### ISSUE TITLE
Event Screen Image Capacity

### ISSUE THREAD

**Issue by JaimeGomezSPE** [krishnandusarkar/SPE-v2#96]
We need to increase the capacity of the Event Screen from 500Kb to 3000Kb.  Is this possible?

**Comment by krishnandusarkar** [comment]
@debasishbanerjee Can this be done easily?

**Comment by snowbreeze** [comment]
Yes. But not recommended to put more than 1MB. @JaimeGomezSPE you can use tinypng.com
@krishnandusarkar I can update this to 1000KB

**Comment by krishnandusarkar** [comment]
@snowbreeze Okay. Do not change anything. I'll take this up with Jaime on today's meeting.
